### DISCUSSION TITLE
github link for download is not working showing 404 not found.

### DISCUSSION THREAD

**Discussion by pankajmamtani** [netbox-community/netbox#12305] [Q&A]
Installation instructions mentioned on the website (https://docs.netbox.dev/en/stable/installation/3-netbox/) not working, please fix and confirm @jeremystretch 

sudo wget https://github.com/netbox-community/netbox/archive/refs/tags/vX.Y.Z.tar.gz
sudo tar -xzf vX.Y.Z.tar.gz -C /opt
sudo ln -s /opt/netbox-X.Y.Z/ /opt/netbox

**Comment by abhi1693** [comment]
Which version did you try downloading?

**Reply by candlerb** [reply]
And what error did you see?

http://www.catb.org/~esr/faqs/smart-questions.html#beprecise

Otherwise we have no idea what problem you're seeing at your side.

**Comment by pankajmamtani** [comment]
Please refer your website link and you may check the version.

It's giving 404 error message.

You can try opening the link on that web which I shared earlier and on
browser also it's not opening and giving same error 404 not found

**Reply by kkthxbye-code** [reply]
You are supposed to replace X, Y and Z with the version number you want to install, or go to the releases page as specified on the page you linked.

There is no version of netbox called X.Y.Z

**Comment by pankajmamtani** [comment]
Can you help aligning with the latest version? And share the URL pls. I was
unable to find any tar file on netbox's github repository.

It's your repo you will know better where that is kept.

**Reply by candlerb** [reply]
Sorry can't tell you that, it's a secret :-)

Hint: go to https://github.com/netbox-community/netbox, then find "Releases" on the right-hand side, and click on the latest release:
![image](https://user-images.githubusercontent.com/44789/233455484-8d2fa78e-1c27-46e2-be1d-be09d2943eb2.png)

The page you then hit has download links for .zip and .tar.gz near the bottom
